openmp/libomptarget/plugins-nextgen/amdgpu/src/rtl.cpp

```diff
@@ -3043,10 +3043,6 @@ struct AMDGPUPluginTy final : public GenericPluginTy {
     // HSA functions from now on, e.g., hsa_shut_down.
     Initialized = true;
 
-#ifdef OMPT_SUPPORT
-    ompt::connectLibrary();
```

Comment:
There is a connectLibrary definition in libomptarget and another in the common plugin. Since the call is now removed from the plugins, the definition in the plugin should also go away. It seems like the entire OmptCallback.cpp can be removed. Don't know whether something breaks then.

Reply:
Ah, well that can be a follow-up patch since it seems to work here. Would be nice to get all of this OMPT stuff out of the plugins, which this should also help since we have actual `init` and `deinit` calls from `libomptarget` now.